### DISCUSSION TITLE
Accidental opening of hover editors

### DISCUSSION THREAD

**Discussion by HEmile** [nothingislost/obsidian-hover-editor#80] [General]
Hi! Loving the plugin :)
I've been using it for a couple weeks now, and one thing that bothers me somewhat is the accidental opening of hover editors. Let me describe a workflow example:
- I'm fairly keyboard heavy, so I open a link under the cursor with the hover editor
- Then, still holding cmd down (all my shortcuts are under cmd), I insert a template
- The problem: If my mouse is somewhere hovering over a link, it'll spawn a new hover editor, even though I'm not using my mouse deliberately to hover over that link!

Outside of being somewhat annoying and taking quite a bit of compute sometimes, I prefer to have the 'auto-pin' option on, which means I have to close these editors down manually too.

I'm posting this as a discussion because I'm not sure what the solution is and whether other people have this issue, or if there's some setting to fix it!

**Comment by huyz** [comment]
I can't replicate this. For me, to start editing, I have to click on the edit Pencil icon, then I have to click where I want to start typing or insert a template.  So I don't know how my mouse pointer can end up hovering over a link.

Maybe a screenshot would help?
